_data/tools.json

```diff
@@ -718,14 +718,23 @@
       "platforms": "Linux",
       "note": "Scans multiple languages for various security flaws. Basically security enhanced code Grep.",
       "type": "SAST"
+   },
+      {
+      "title": "HCL AppScan CodeSweep - GitHub Action",
+      "url": "https://hclsw.co/codesweep",
+      "owner": "HCL Sofware",
+      "license": "Open Source or Free",
+      "platforms": null,
+      "note": "Scan the new code on a push/pull request using a GitHub action. Findings are highlighted in the `Files Changed` view and details about the issue and mitigation steps can be found in the `Actions` page. Unrestricted usage allowed with a free trial account. The tool currently supports Python, Ruby, JS (Vue, React, Node, Angular, JQuery, etc), PHP, Perl, COBOL, APEX & a few more.",
+      "type": "SAST"
    },
    {
-      "title": "HCL AppScan CodeSweep",
+      "title": "HCL AppScan CodeSweep - VS Code Plugin",
       "url": "https://hclsw.co/codesweep",
-      "owner": null,
+      "owner": "HCL Software",
       "license": "Open Source or Free",
```

Comment:
Really?

Reply:
I'll close this one and make some corrections.